cli/cliutils/cliutils.go

```diff
@@ -1263,6 +1265,35 @@ func SetDefaultArch() {
 	}
 }
 
+//Get and veify public key file
+func GetAndVerifyPublicKey(pubKeyFilePath string) string {
+	msgPrinter := i18n.GetMessagePrinter()
+	msgPrinter.Printf("Verifying public key file ... ")
+	msgPrinter.Println()
+	pubKeyFilePath_tmp := WithDefaultEnvVar(&pubKeyFilePath, "HZN_PUBLIC_KEY_FILE")
+	pubKeyFilePath = VerifySigningKeyInput(*pubKeyFilePath_tmp, true)
+
+	//inBytes := cliutils.ReadFile(pubKeyFilePath)
```

Comment:
Please remove the commented out lines here and below

Reply:
comments removed as requested.